cmd/crates/soroban-rpc/README.md

```diff
@@ -0,0 +1,3 @@
+# soroban-rpc
+
+Tools and utilities for soroban rpc.
```

Comment:
Same comment above; this should live in `soroban-rpc` eventually but it's fine for now